march_rqt_gait_generator/src/march_rqt_gait_generator/gait_generator_controller.py

```diff
@@ -35,6 +38,14 @@ def connect_buttons(self):
         self.view.import_gait_button.clicked.connect(self.import_gait)
         self.view.export_gait_button.clicked.connect(self.export_gait)
 
+        self.view.import_previous_subgait_button.clicked.connect(lambda: self.import_side_subgait('previous'))
+        # self.view.previous_is_standing_check_box.stateChanged.connect(self.toggle_previous_is_standing)
+        self.view.lock_startpoint_check_box.stateChanged.connect(self.toggle_lock_startpoint)
+
+        self.view.import_next_subgait_button.clicked.connect(lambda: self.import_side_subgait('next'))
+        # self.view.previous_is_standing_check_box.stateChanged.connect(self.toggle_next_is_standing)
+        self.view.lock_endpoint_check_box.stateChanged.connect(self.toggle_lock_endpoint)
```

Comment:
You should remove these commented lines

Reply:
Isn't this necessary for the second part of the issue?